<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogSetupRadialSheet</class>
 <widget class="QDialog" name="dialogSetupRadialSheet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>198</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <widget class="QSpinBox" name="spinFields">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinPoints">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество полей</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество точек</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
